examples/source/user-consent/Server_Side_User_Consent_Flow.html

```diff
@@ -0,0 +1,121 @@
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    :root {
+      --space-2: 1rem;   /* 16px */
+      --space-3: 1.5rem; /* 24px */
+    }
+    .consentPopup {
+      padding: var(--space-3);
+      margin: 0 auto;
+      background: #fff;
+      border-radius: 5px;
+      position: relative;
+      max-width: 700px;
+      width: 95%;
+    }
+    .consentPopup > h2, /* overwrite ABE style */
+    .consentPopup > * {
+      margin: 0;
+      margin-bottom: var(--space-2);
+    }
+    .consentPopup > button {
+      margin-right: var(--space-2);
+    }
+    .popupOverlay {
+      height: 100vh;
+      width: 100vw;
+      background: rgba(0, 0, 0, 0.7);
+      padding: 5% 0;
+    }
+    .dismiss-button {
+      position: absolute;
+      right: var(--space-3);
+      top: var(--space-2);
+      cursor: pointer;
+    }
+    #post-consent-ui {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: var(--space-3);
+    }
+  </style>
+</head>
+<body>
+
+  <!-- ## Defining the consent flow -->
+  <!--
+    We define a basic server-side consent flow, similar to [this sample](/content/amp-dev/documentation/examples/documentation/Client_Side_User_Consent_Flow.html). The flow should only trigger when the servers response that `consentRequired` is true and there is not `consentStateValue` or `consentString` for this user.
+
+    When a user lands on the AMP page, amp-consent will check local storage for an existing consent state. 
+    If a consent state exists, it will unblock all components. Otherwise, it will wait for the response from `checkConsentHref` to determine if consent is required for this user and if there is an existing consent state. If there is an existing consent state, amp-consent will use that value and unblock. Otherwise, it will show the promptUi (if configured).
+
+    Use `expireCache: true`, in your response to never save consent states to local storage and thus always prompt servers for consent state.
+
+    A recommended design for sites that can manage their own consent (via external page) is to always respond with: 
+    ```json
+    {
+        "expireCache": true,
```

Comment:
Hmmm, we would not recommend setting `expireCache: true` due to performance issue. The best approach would be to use `expireCache: true` only when there's need to erase cached value.

Reply:
Good point, removed the expireCache response, and instead explained expireCache in a Tip